plugins/outputs/pagerduty/pagerduty.go

```diff
@@ -0,0 +1,114 @@
+package pagerduty
+
+import (
+	"fmt"
+	pd "github.com/PagerDuty/go-pagerduty"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"go/token"
+	"go/types"
+	"log"
+)
+
+type PD struct {
+	ServiceKey string            `toml:"service_key"`
+	Desc       string            `toml:"description"`
+	Metric     string            `toml:"metric"`
+	Field      string            `toml:"field"`
+	Expression string            `toml:"expression"`
+	Tags       map[string]string `toml:"tags"`
```

Comment:
Calling this map just "tags" won't work because telegraf will interpret this map as tags that should be added to the metrics. Maybe call it "TagFilter" and provide an example of how to use it in SampleConfig.